### PR TITLE
feat: Add model_monitoring_v2 notebooks

### PR DESCRIPTION
**Pull request by SereniCode**
Add three notebooks to model_monitoring_v2:

- model_monitoring_for_custom_model_batch_prediction_job.ipynb
- model_monitoring_for_model_outside_vertex.ipynb
- model_monitoring_for_custom_model_online_prediction.ipynb

<br>
feat: Add model_monitoring_v2 notebooks
<br><br><br>

**REQUIRED:** Fill out the below checklists or remove if irrelevant
1. If you are opening a PR for `Official Notebooks` under the [notebooks/official](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/official) folder, follow this mandatory checklist:
- [ ] Use the [notebook template](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/notebook_template.ipynb) as a starting point.
- [ ] Follow the style and grammar rules outlined in the above notebook template.
- [ ] Verify the notebook runs successfully in Colab since the automated tests cannot guarantee this even when it passes.
- [ ] Passes all the required automated checks. You can locally test for formatting and linting with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).
- [ ] You have consulted with a tech writer to see if tech writer review is necessary. If so, the notebook has been reviewed by a tech writer, and they have approved it.
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/official/CODEOWNERS) file under the `Official Notebooks` section, pointing to the author or the author's team.
- [ ] The Jupyter notebook cleans up any artifacts it has created (datasets, ML models, endpoints, etc) so as not to eat up unnecessary resources.

<br>

2. If you are opening a PR for `Community Notebooks` under the [notebooks/community](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/notebooks/community) folder:
- [ ] This notebook has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/notebooks/community/CODEOWNERS) file under the `Community Notebooks` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

<br>

3. If you are opening a PR for `Community Content` under the [community-content](https://github.com/GoogleCloudPlatform/vertex-ai-samples/tree/main/community-content) folder:
- [ ] Make sure your main `Content Directory Name` is descriptive, informative, and includes some of the key products and attributes of your content, so that it is differentiable from other content
- [ ] The main content directory has been added to the [CODEOWNERS](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/community-content/CODEOWNERS) file under the `Community Content` section, pointing to the author or the author's team.
- [ ] Passes all the required formatting and linting checks. You can locally test with these [instructions](https://github.com/GoogleCloudPlatform/vertex-ai-samples/blob/main/CONTRIBUTING.md#code-quality-checks).

### CODE REVIEW

File: notebooks/official/model_monitoring_v2/model_monitoring_for_custom_model_batch_prediction_job.ipynb [hunk 1]
```diff
@@ -0,0 +1,1279 @@
+{
```
Comment (gericdong) [inline notebooks/official/model_monitoring_v2/model_monitoring_for_custom_model_batch_prediction_job.ipynb:1]: <p>Update the link to Github.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2988/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (gericdong) [inline notebooks/official/model_monitoring_v2/model_monitoring_for_custom_model_batch_prediction_job.ipynb:1]: <p>.. how to use with the Vertex AI SDK for Python to set up Vertex AI Model Monitoring V2 for batch predictions.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2988/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: notebooks/official/model_monitoring_v2/model_monitoring_for_custom_model_online_prediction.ipynb [hunk 2]
```diff
@@ -0,0 +1,1545 @@
+{
```
Comment (gericdong) [inline notebooks/official/model_monitoring_v2/model_monitoring_for_custom_model_online_prediction.ipynb:1]: <p>Update the link to Github</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2988/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (gericdong) [inline notebooks/official/model_monitoring_v2/model_monitoring_for_custom_model_online_prediction.ipynb:1]: <p>..how to use with the Vertex AI SDK for Python to set up Vertex AI Model Monitoring V2 for online predictions.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2988/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: notebooks/official/model_monitoring_v2/model_monitoring_for_model_outside_vertex.ipynb [hunk 3]
```diff
@@ -0,0 +1,1220 @@
+{
```
Comment (gericdong) [inline notebooks/official/model_monitoring_v2/model_monitoring_for_model_outside_vertex.ipynb:1]: <p>Update the link to Github.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2988/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Comment (gericdong) [inline notebooks/official/model_monitoring_v2/model_monitoring_for_model_outside_vertex.ipynb:1]: <p>Vertex AI Python client library &gt; Vertex AI SDK for Python</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2988/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

File: notebooks/official/model_monitoring_v2/model_monitoring_for_custom_model_batch_prediction_job.ipynb (hunk 1 above)
Comment (SereniCode) [reply notebooks/official/model_monitoring_v2/model_monitoring_for_custom_model_batch_prediction_job.ipynb:1]: Done

**Review by gericdong — APPROVED** [review]

**Pull request merged**